apps/condo/domains/organization/schema/OrganizationEmployee.js

```diff
@@ -141,6 +141,13 @@ const OrganizationEmployee = new GQLListSchema('OrganizationEmployee', {
             if (!hasDbFields(['organization'], resolvedData, existingItem, context, addValidationError)) return
             if (!hasOneOfFields(['email', 'name', 'phone'], resolvedData, existingItem, addValidationError)) return
         },
+        afterChange: async ({ context, operation, existingItem, updatedItem }) => {
+            const isSoftDeleteOperation = operation === 'update' && !existingItem.deletedAt && Boolean(updatedItem.deletedAt)
+
+            if (isSoftDeleteOperation) {
+                await softDeletePropertyScopeOrganizationEmployee(context, updatedItem)
```

Comment:
we can't avoid this :( we need some tools for that. Can you mark this line as tech dept?